README.md

```diff
@@ -1,46 +1,118 @@
-Then spin up Nginx:
+To get dojot running with http on a domain other than localhost, at least for the purposes of this guide, you **MUST** ensure you have set up a static public IP address for your server and registered a domain for it. Then just follow the next steps. *And that domain must be accessible where dojot is running, an alternative solution for this is look for the commented out codes in `docker-compose.yml` preceded by the titles `Gateway-static-dojot _default-1`, `Gateway-static-dojot_default-2` and `Gateway-static-dojot_default-3` and uncomment the codes accordingly instructions.*
+
+Add the following lines to the end of the file [.env](./.env) and change `<your domain>` for your registered domain. (If you want to use port 8000 do not add the variable `DOJOT_HTTP_PORT`, It be accessible at `http://<your domain>:8000`.)
 
 ``` sh
-# Go to the repository letsencrypt-nginx
-cd letsencrypt-nginx
-# Start the Nginx container
-sudo docker-compose -f docker-compose-challenge.yml up -d
+DOJOT_DOMAIN_NAME=<your domain>
+
+DOJOT_HTTP_PORT=80
+
+DOJOT_URL=http://${DOJOT_DOMAIN_NAME}:${DOJOT_HTTP_PORT}
+
 ```
 
-The next step is to run Certbot tool. **Change the command bellow to use your
-email address and your domain, and just run it**.
+Then run the command below:
+
+``` sh
+sudo docker-compose up  -d
+```
+
+After that the dojot must be accessible at `http://<your domain>`. The tenant will be the value of the `KEYCLOAK_DEFAULT_REALM` variable which by default has the value `admin`, the username will be `admin` and the password the value defined in `KEYCLOAK_ADMIN_PASSWORD_TEMP`.
+
+## How to run with HTTPS (secure dojot with Let's Encrypt) - recommended
+
+To get dojot running with https, at least for the purposes of this guide, you **MUST** ensure you have set up a static public IP address for your server and registered a domain for it. Then just follow the next steps. *And that domain must be accessible where dojot is running, , an alternative solution for this is look for the commented out codes in `docker-compose.yml` preceded by the titles `Gateway-static-dojot _default-1`, `Gateway-static-dojot_default-2` and `Gateway-static-dojot_default-3` and uncomment the codes accordingly instructions.*
+
+Add the following lines to the end of the file [.env](./.env) and change `<your domain>` for your registered domain.
 
 ``` sh
- sudo docker run -it --rm \
+DOJOT_DOMAIN_NAME=<your domain>
+
+DOJOT_HTTP_PORT=80
+DOJOT_HTTPS_PORT=443
+
+DOJOT_ENABLE_HTTPS_ONLY=true
+
+DOJOT_URL=https://${DOJOT_DOMAIN_NAME}:${DOJOT_HTTPS_PORT}
+
+KEYCLOAK_REALM_SSL_MODE=EXTERNAL
+```
+
+Then change the file `letsencrypt-nginx/nginx-challenge.conf`, replacing __`<YOUR DOMAIN>`__ tag by your registered domain.
+
+After changing `letsencrypt-nginx/nginx-challenge.conf`, run the command below to run the service that will be used by Let's Encrypt to verify your registered domain.
+
+``` sh
+sudo docker-compose up -d letsencrypt-nginx
```

Comment:
is not it necessary inform docker-compose-challenge.yml ?

Reply:
Not anymore